rpi-setup.sh

```diff
@@ -109,6 +109,10 @@ cat > $vesselJson <<jsonfile
 }
 jsonfile
 
+sudo chmod 644 $vesselJson
+sudo chown pi $vesselJson
```

Comment:
```
tmp tjk$ sudo chown root foo.txt
tmp tjk$ ls -l foo.txt 
-rw-r--r--  1 root  wheel  1 Dec  1 10:49 foo.txt
tmp tjk$ sudo chown $USER foo.txt
tmp tjk$ ls -l foo.txt 
-rw-r--r--  1 tjk  wheel  1 Dec  1 10:49 foo.txt
```
As I understand it $USER is evaluated before calling sudo.
Another method would be to use a variable: assign it, then use it on the sudo line.